derp/derp_server.go

```diff
@@ -1824,6 +1852,18 @@ func (c *sclient) sendPacket(srcKey key.NodePublic, contents []byte) (err error)
 	return err
 }
 
+// noteSendFromSrc notes that sclient is about to be written
+// a packet from src.
```

Comment:
Nitpick, but: this is a tricky sentence to read; how about:
```suggestion
// noteSendFromSrc notes that we are about to write a packet
// from src to sclient.
```

?

Reply:
done